src/instructions.html

```diff
@@ -1,46 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Luckiest+Guy&family=Roboto:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
-    <link rel="stylesheet" href="instructions.css" />
-    <title>WAR - How to Play</title>
-  </head>
-  <body>
-    <div class="container">
-        <a href="#">
-          <img src="../public/gearIcon.png" alt="settings gear icon" id="settings">
-        </a>
-        <h1>
-          War!
-        </h1>
-        <div class="intructions">
-            <h2>
-              How to Play
-            </h2>
-            <p>
-              The game begins with the deck being split evenly between you and the other player, or computer. At the top of the round, each player will tap the top of their respective decks to play the top card. Playing the cards turns them face up revealing the face value of the cards. Whichever card has the higher face value wins and the winner takes both cards and they are placed face down at the bottom of the winning player's deck.
-            </p>
-            <h3 id="question-same">
-              But what if both cards hold the same face value?
-            </h3>
-            <p>
-              Great question! In the event of a tie, both players enter into War! mode. In War! mode, each player taps their deck to play one card face down and one card face up. If either of the cards is a higher face value, the winning player takes all cards currently in play and adds them to their deck. If another tie happens, War! mode continues until one player has one the pile of cards in play.
-            </p>
-            <h3 id="question-win">
-              How do I win?
-            </h3>
-            <p>
-              Another great question! The first player to collect all 52 cards in their deck wins the game. 
-            </p>
-            <p>Sounds easy, right? But don't get too confident! After all, you may win a few battles, but can you win the War?</p>
-        </div>
-        <a href="#" id="play">Play Now</a>
-        <a href="#" id ="home">Back to Home</a>
+
```

Comment:
Would it be safe to delete this html file?

Reply:
Yeah, the instructions.html file is no longer needed.